test_python/test_c2.py

```diff
@@ -17,8 +17,9 @@
 import numpy as np
 import hypothesis.strategies as st
 import caffe2.python.hypothesis_test_util as hu
+import tensor_comprehensions as tc
```

Comment:
These changes I have tested within fbcode and things worked there.
Note the perf aspects in this test dependent on #327 being reviewed and landed.